apps/spruce/src/components/PatchesPage/RequesterSelector.tsx

```diff
@@ -0,0 +1,59 @@
+import Dropdown from "components/Dropdown";
+import { TreeSelect } from "components/TreeSelect";
+import { noFilterMessage } from "constants/strings";
+import { useStatusesFilter } from "hooks";
+import { PatchPageQueryParams } from "types/patch";
+
+export const RequesterSelector: React.FC = () => {
+  const { inputValue: statusVal, setAndSubmitInputValue: statusValOnChange } =
+    useStatusesFilter({ urlParam: PatchPageQueryParams.Requesters });
+
+  return (
+    <Dropdown
```

Comment:
Would you be opposed to using [Combobox](https://www.mongodb.design/component/combobox/example/) instead of our custom component?